### ISSUE TITLE
I consider redoing this for other languages, is this possible for a private person?

### ISSUE THREAD

**Issue by stefangrotz** [tatsu-lab/stanford_alpaca#124]
I would love to recrate this experiment in German and it looks easy enough, I bascially just have to adapt [prompt.txt](https://github.com/tatsu-lab/stanford_alpaca/blob/main/prompt.txt) and follow the rest of the instructions, right?

I am willing to invest arount 100€ into this, do you think it could be feasible? I don't care if the training is slow or if I need a few months for it. So do you see more ways to optimize the process?

**Comment by yrvelez** [comment]
This project created a Portuguese version for $8 simply by translating the instruction following dataset: https://github.com/22-hours/cabrita/blob/main/README.md. Assuming similar translation costs for German, it sounds feasible to me. As for training, the LoRa version of Alpaca can be trained quickly in Google Colab for free.

**Comment by stefangrotz** [comment]
Yes, I will document my experiments in the readme of this repo: https://github.com/stefangrotz/alman-alpaca

Anyone is welcome to participate.

**Comment by Whadup** [comment]
We're training models on German instructions over here: https://github.com/Whadup/llamarr_alpaca/

**Comment by stefangrotz** [comment]
Great, than I'll join you and delete my repo.

**Comment by Whadup** [comment]
This is just the JSON standard way of storing unicode. If you load it with python, you'll get nice unicode text. 
You could also store it as unicode-json with `json.dump(data, file, ensure_ascii=False)`

**Comment by stefangrotz** [comment]
Good to know :) 

I have a few ideas about creating adidtional training data in German using Wikidata, Wikipedia, Wikihow and advanced GPT-4 prompts. Can I just join the discussion on your reopo for this?

**Comment by stefangrotz** [comment]
@Whadup I just started the training of my first model using the translation from your repo. Thanks for sharing it, I plan to share my model in the coming days.

I plan to translate https://github.com/gururise/AlpacaDataCleaned into German, but it might be worth waiting a few weeks, since people are still cleaning up the dataset in this repo.

**Comment by Whadup** [comment]
Hi,
yeah I wasn't sure about how to share the model, but if you send me a PM I think I could share my checkpoint.
Using the cleaned dataset sounds good, I still have my deepl-script for translating. 

btw, translating with deepl destroys all tasks of the form "fix the spelling/grammar/style of the following input.", as deepl already fixes everything in the input. So I guess this needs manual work after all
